ui/components/app/snaps/snap-update-alert/snap-update-alert.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BannerAlert, IconName, Text } from '../../../component-library';
+import { TextColor } from '../../../../helpers/constants/design-system';
+
+const SnapUpdateAlert = ({ snapName, onUpdateClick, bannerAlertProps }) => {
+  return (
+    <BannerAlert
+      title="Update available"
+      actionButtonLabel="Update"
+      actionButtonOnClick={onUpdateClick}
+      actionButtonProps={{
+        endIconName: IconName.Download,
+        color: TextColor.primaryDefault,
+      }}
+      {...bannerAlertProps}
+    >
+      <Text>Get the latest version of {snapName}</Text>
```

Comment:
All of the text in this file should be translatable

Reply:
Thanks for noticing! I forgot about these. Fixed now.